webcompat/form.py

```diff
@@ -59,7 +59,7 @@
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 browser_test_label = u'Did you test in another browser?'
 
-steps_default = u"""For example,
+steps_default = u"""\nFor example,
```

Comment:
It's better if we add the newline in the actual stringdoc... this can be removed. Will comment below.